src/deployCommand.ts

```diff
@@ -259,3 +271,26 @@ export const getCoverageFormattersOptions = (formatters: string[] = []): Coverag
     reportOptions,
   };
 };
+
+export const targetUsernameFlag = Flags.string({
```

Comment:
Having target-org with short char other than 'o' break sf guidelines. I ran into short char collision in plugin-user and to avoid the collision we ended up retaining the old commands that used the `-o` and created new ones that allows style adherence.